ui/templates/base.html

```diff
@@ -120,5 +120,14 @@
     {% endcompress %}
     {% block postcontent %}
     {% endblock %}
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new
+      Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+      ga('create', '{{ settings.GOOGLE_ANALYTICS_ID }}', 'auto');
+      ga('send', 'pageview');
+    </script>
```

Comment:
Nit: Should this be maybe placed into a block that only renders when the `GOOGLE_ANALYTICS_ID` setting is set?